zone/mob.cpp

```diff
@@ -3856,6 +3937,61 @@ int16 Mob::GetHealRate(uint16 spell_id, Mob* caster) {
 	return heal_rate;
 }
 
+void Mob::SetBottomRampageList()
```

Comment:
These implementations I think are a bit subpar. We should be able to add hatelist functions and the mob will call them and do it without the whole entity list bit.